bionic/dagviz.py

```diff
@@ -155,6 +155,7 @@ def doc_from_node(node):
             )
             if doc:
                 tooltip = rewrap_docstring(doc)
+                tooltip += ", persist = " + str(graph.nodes[node].get("should_persist", ""))
```

Comment:
The `doc` string will probably be in complete sentences, so attaching a comma to the end may not look right. Maybe something like this?

    rewrap_docstring(doc) + f"\n\n Persisted: {should_persist}"

Also, what should we do when `doc` is not present?